5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf

```diff
@@ -15,13 +15,14 @@
 locals {
   cloud_build_sas = ["serviceAccount:${google_service_account.cloud_build.email}"] # cloud build service accounts used for CI
   membership_re   = "projects/([^/]*)/locations/([^/]*)/memberships/([^/]*)$"
+  gke_projects    = [regex(local.membership_re, var.cluster_membership_id_dev)[0], regex(local.membership_re, var.cluster_membership_ids_nonprod[0])[0], regex(local.membership_re, var.cluster_membership_ids_prod[0])[0]]
```

Comment:
For the future when parameterizing ENVs: probably best to wrap in `distinct()` to support multiple ENVs in a single project

Reply:
Can incorporate into https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/154